idx/memory/meta_tags_test.go

```diff
@@ -380,3 +262,26 @@ func TestDeletingMetaRecord(t *testing.T) {
 		t.Fatalf("Expected returned record id to not be present, but it was")
 	}
 }
+
+func metaTagRecordsAreEqual(record1, record2 *tagquery.MetaTagRecord) bool {
```

Comment:
we should really just standardize on `<type>.Equals` methods?

Reply:
Yeah, I think we'll actually need two of those `.Equals` methods on the meta tag record type. One is comparing the whole meta tag record, the other is comparing only the expressions. 
So I'll name them `.Equals` and `.EqualExpressions`.